pkg/roachprod/install/cluster_synced.go

```diff
@@ -1421,6 +1421,13 @@ func (c *SyncedCluster) Wait(ctx context.Context, l *logger.Logger) error {
 			}
 			res.Err = errors.Wrapf(res.Err, "timed out after 5m")
 			l.Printf("  %2d: %v", node, res.Err)
+			logContent, err := c.getLogTail(ctx, node, 20)
+			if err != nil {
+				l.Printf("  %2d: %v (failed to retrieve log tail: %v)", node, res.Err, err)
```

Comment:
`res.Err` has already been logged above; it's unaffected by the `getLogTail` call.